tests/integration/relations/test_db.py

```diff
@@ -213,6 +215,7 @@ async def test_relation_with_indico(ops_test: OpsTest):
     )
 
 
+@pytest.mark.unstable
 async def test_relation_with_openldap(ops_test: OpsTest):
```

Comment:
The openldap consistently fails, in other PRs as well. Created [DPE-2685](https://warthogs.atlassian.net/browse/DPE-2685) to investigate.

Reply:
IMHO, it was a Canonical registry migration, should no longer be reproducible. :crossed_fingers: